codec/src/main/java/io/netty/handler/codec/compression/Lz4FrameEncoder.java

```diff
@@ -115,7 +118,7 @@ public Lz4FrameEncoder() {
      */
     public Lz4FrameEncoder(boolean highCompressor) {
         this(LZ4Factory.fastestInstance(), highCompressor, DEFAULT_BLOCK_SIZE,
-                XXHashFactory.fastestInstance().newStreamingHash32(DEFAULT_SEED).asChecksum());
+                XXHashFactory.fastestInstance().newStreamingHash32(DEFAULT_SEED).asChecksum(), DEFAULT_MAX_ENCODE_SIZE);
```

Comment:
remove the new argument? Delegating to the other constructor should supply `DEFAULT_MAX_ENCODE_SIZE` for us in a central location.